homework2/script1.js

```diff
@@ -0,0 +1,21 @@
+const chingachung = (p1, p2) => {
+  switch (true) {
+    case p1 === p2:
+      return "It's a draw";
+    case p1 === "Rock" && p2 === "Scissors":
```

Comment:
you have repetitive string values here. It's more common to have in one variable and check values from there. Here is an example:
`const ROCK = 'rock'
const SCISSORS = 'scissors'
const PAPER = 'paper'`